s2fft/precompute_transforms/construct.py

```diff
@@ -53,27 +60,102 @@ def spin_spherical_kernel(
             "Reality acceleration only supports spin 0 fields. "
             + "Defering to complex transform."
         )
+    if mode.lower() == "price_mcewen" and abs(spin) >= 5:
+        mode = "risbo"
+        warn(
+            "The Price-McEwen can become unstable for spins >= 5. "
+            + "Defaulting to slower but more stable Risbo recursions."
+        )
+
+    if mode.lower() == "auto":
+        # This mode automatically determines which recursion is best suited for the
+        # current parameter configuration.
+        mode = "risbo" if abs(spin) >= 5 else "price_mcewen"
```

Comment:
Can we make the 5 a global variable that we pick up whenever we do this check.  Then if we want to change this we just need to make an update one place in the code.

Reply:
Introduced a global variable PM_MAX_STABLE_SPIN which is called throughout.